extensions/applicationinsights-dependencies-js/src/ajax.ts

```diff
@@ -555,83 +552,88 @@ export class AjaxMonitor extends BaseTelemetryPlugin implements IDependenciesPlu
+                                                let ajaxResponse:IAjaxRecordResponse = {
+                                                    statusText: response.statusText,
+                                                    headerMap: null,
+                                                    correlationContext: _getFetchCorrelationContext(response)
+                                                };
+        
+                                                if (_enableResponseHeaderTracking) {
+                                                    const responseHeaderMap = {};
+                                                    response.headers.forEach((value: string, name: string) => {     // @skip-minify
+                                                        if (_canIncludeHeaders(name)) {
+                                                            responseHeaderMap[name] = value;
+                                                        }
+                                                    });
+        
+                                                    ajaxResponse.headerMap = responseHeaderMap;
+                                                }
+        
+                                                return ajaxResponse;
+                                            });
+        
+                                            return response;
+                                        })
+                                            .catch((reason: any) => {
+                                                _reportFetchMetrics(callDetails, 0, input, null, fetchData, null, { error: reason.message });
+                                                throw reason;
+                                            });
+                                    }
                                 }
+                            },
+                            // Create an error callback to report any hook errors
+                            hkErr: _createErrorCallbackFunc(_self, _eInternalMessageId.FailedMonitorAjaxOpen,
+                                "Failed to monitor Window.fetch" + ERROR_POSTFIX)
+                        }));
+    
+                        _fetchInitialized = true;
+                    } else if (isPolyfill) {
+                        // If fetch is a polyfill we need to capture the request to ensure that we correctly track
+                        // disabled request URLS (i.e. internal urls) to ensure we don't end up in a constant loop
+                        // of reporting ourselves, for example React Native uses a polyfill for fetch
+                        // Note: Polyfill implementations that don't support the "poyyfill" tag are not supported
+                        // the workaround is to add a polyfill property to your fetch implementation before initializing
+                        // App Insights
+                        _addHook(InstrumentFunc(global, strFetch, {
```

Comment:
This could also get called more than once when the referenced config items are called more than once.